### PR TITLE
Fixes FreeRDP/Remmina#473 - Customize button in the connection editor should  show the resolutions dialog

### PR DESCRIPTION
**Pull request by antenore**
@muflone it works, but I'm not sure about the quality of the code. Can you please review it?

Thanks in advance

### CODE REVIEW

File: remmina/src/remmina_file_editor.c [hunk 1]
```diff
@@ -428,6 +429,53 @@ static void remmina_file_editor_create_password(RemminaFileEditor* gfe, GtkWidge
...
+		}
+	}
+
+	if (width == 0 || height == 0)
+		result = FALSE;
+
+	if (!result)
+		*error = g_strdup(_("Please enter format 'widthxheight'."));
+	return result;
+}
```
Comment (muflone) [inline remmina/src/remmina_file_editor.c]: This thing, duplicated between preferences dialog and file editor could be moved in remmina_public to avoid code duplication.

File: remmina/src/remmina_file_editor.c [hunk 2]
```diff
@@ -476,10 +531,10 @@ static void remmina_file_editor_create_resolution(RemminaFileEditor* gfe, const
...
 	resolution = remmina_file_get_string(gfe->priv->remmina_file, "resolution");
 
-	widget = remmina_public_create_combo_text_d(remmina_pref.resolutions, resolution, NULL);
-	gtk_widget_show(widget);
-	gtk_box_pack_start(GTK_BOX(hbox), widget, TRUE, TRUE, 0);
-	gfe->priv->resolution_custom_combo = widget;
+	combo = remmina_public_create_combo_text_d(remmina_pref.resolutions, resolution, NULL);
+	gtk_widget_show(combo);
+	gtk_box_pack_start(GTK_BOX(hbox), combo, TRUE, TRUE, 0);
+	gfe->priv->resolution_custom_combo = combo;
```
Comment (muflone) [inline remmina/src/remmina_file_editor.c]: Why do you create another widget reference?
Is this change really needed?

**Comment by muflone** [comment]
This is great, @antenore 
Could you please note the previous comments above?

**Comment by antenore** [comment]
Thanks @muflone . I'll work on it the next days.

**Comment by muflone** [comment]
@antenore 
I'm going to make some changes to the string_list code, including the remmina_string_list_new.
Could you please make the previous changes and commit or you will need to solve a conflict

**Comment by antenore** [comment]
Don't worry @muflone , go ahead, . I'll solve the conflicts.

I still cannot work on Remmina for some days.

**Comment by antenore** [comment]
@muflone , @giox069 

Back to business, can you kindly take a look please.

**Comment by muflone** [comment]
Seems fine to me, you can merge it
